### ISSUE TITLE
How can i change the batch-size?

### ISSUE THREAD

**Issue by SecondLi** [CoinCheung/DeepLabv3plus#8]
How can i change the batch-size?

**Comment by CoinCheung** [comment]
you can configure the batch size and the associated learning rate and training iters in the config file. It is easy to find :)
